youtube_dl/extractor/nick.py

```diff
@@ -85,7 +85,8 @@ class NickBrIE(MTVServicesInfoExtractor):
                     https?://
                         (?:
                             (?P<domain>(?:www\.)?nickjr|mundonick\.uol)\.com\.br|
-                            (?:www\.)?nickjr\.[a-z]{2}
+                            (?:www\.)?nickjr\.[a-z]{2}|
+                            (?:www\.)?nickelodeonjunior\.[a-z]{2}
```

Comment:
Add tests for other www.nickelodeonjunior.X domains.

Reply:
After checking on Wikipedia, only France seem to use that specific domain, all the other use nickjr.[a-z]{2}

Made a commit to only support the .fr extension